internal/cmd/egctl/version.go

```diff
@@ -75,10 +77,16 @@ func versions(w io.Writer, containerName, output string) error {
 
 	pods, err := c.PodsForSelector(metav1.NamespaceAll, "control-plane=envoy-gateway")
 	if err != nil {
-		return fmt.Errorf("list EG pods failed: %w", err)
+		return errors.Wrap(err, "list EG pods failed")
 	}
 
 	for _, pod := range pods.Items {
+		if pod.Status.Phase != "Running" {
```

Comment:
what about the case when no pods are running ? we will just skip this entire section, shouldnt we add some helpful prompt for the user when such a case happens ?

Reply:
I think we should print some warning message in that case.

Reply:
this is fine for now, but in the future we should be try and remove any kubeism from the command